sys/vane/jael.hoon

```diff
@@ -852,9 +937,9 @@
   ::  urb: absolute urbit state
   ::  sub: relative urbit state
   ::
-  =*  urb  ->-
-  =*  sub  ->+<
-  =*  etn  ->+>
+  =*  urb  ->+<
+  =*  sub  ->+>-
+  =*  etn  ->+>+
```

Comment:
Every time I see a PR touch this I cringe. This feels pretty stupid, but I guess there's no better way of having both direct access to the contents, and a face for reference of the whole?

Reply:
It'd probably be possible to achieve the desired affect in clearer ways. This vane predates `=,` (and `+*` in `|_`).